### ISSUE TITLE
[BUG] playAsset and loadAsset are missing from index.d.ts

### ISSUE THREAD

**Issue by RenatoFerracini** [johnsonsu/react-native-sound-player#170]
Without it, typescript shows an error in code editor. I only added these two lines to SoundPlayerType and worked:

```
    playAsset: (asset: number) => void;
    loadAsset: (asset: number) => void;
```

**Comment by pke** [comment]
@johnsonsu could be included in the next release?

**Comment by johnsonsu** [comment]
Fixed in [0.14.3](https://github.com/johnsonsu/react-native-sound-player/releases/tag/0.14.3).

**Comment by pke** [comment]
Thanks that was quick!
